utils/jobstats/jobstats.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+#
+# ==-- jobstats - support for reading the contents of stats dirs --==#
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014-2017 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ==------------------------------------------------------------------------==#
+#
+# This file contains subroutines for loading object-representations of one or
+# more directories generated by `swiftc -stats-output-dir`.
+
+import datetime
+import itertools
+import json
+import os
+import random
+import re
+
+
+class JobStats:
```

Comment:
Why use old style classes?

Reply:
I don't know about new-style classes. Also I have no idea which python any given style of classes is compatible with. I figured 'class X' has got to be the most widely used?

Reply:
Info: https://wiki.python.org/moin/NewClassVsClassicClass

Just do:

```
class JobStats(object):
```

and your good.

Reply:
Done. Thanks!